src/vivarium_nih_us_cvd/constants/data_values.py

```diff
@@ -107,8 +109,8 @@ def name(self):
 class __SBPThreshold(NamedTuple):
     """sbp exposure thresholds"""
 
-    LOW: int = 130
-    HIGH: int = 140
+    LOW: float = 130
+    HIGH: float = 140
 
```

Comment:
Why change to float here? You were getting warnings on compares of float and int?

Reply:
No. I think just logically those should be float values